index.css

```diff
@@ -0,0 +1,37 @@
+#main{
+    background-color: #EEEEEE;
+    padding: 15px;
+}
+
```

Comment:
I would not recommend adding spaces between selectors. This could be a source linters errors, especially stylelinters.